README.md

```diff
@@ -73,3 +73,26 @@ Once you have installed SimpleSAMLphp, installing this module is very simple. Fi
 ```
 'instance_name' => 'Instance name',
 ```
+
+### Writing via API
+#### Configuration
+Add the following (and adjust the credentials) to enable writing via the API (example request following). Methods supported are `POST,PUT`.
+```
+  'apiWriteEnabled' => true,
+  'apiWriteUsername' => 'api_writer,
+  'apiWritePassword' => 'ap1Wr1T3rP@S$,
+```
+#### Example request
+```
+curl --request POST \
+  --url https://proxy.com/proxy/module.php/proxystatistics/writeLoginApi.php \
+  --header 'Authorization: Basic encodedCredentials' \
```

Comment:
basic auth? 😟

Reply:
How about OIDC or [JWT](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-jwt-bearer)? (aka `client_secret_jwt` or `private_key_jwt` in OIDC)

Reply:
Will check with LS Proxy team if they would be able to implement it like that ;)